packages/dds/tree/src/simple-tree/proxyBinding.ts

```diff
@@ -44,7 +43,7 @@ const proxyToAnchorNode = new WeakMap<TreeNode, AnchorNode>();
  * The raw node exists when the proxy is first created but before it has been associated with a real {@link FlexTreeNode}.
  * For example, after a user calls `const proxy = new Foo({})` but before `proxy` is inserted into the tree and queried.
  */
-const proxyToRawFlexNode = new WeakMap<TreeNode, RawTreeNode<FlexFieldNodeSchema, unknown>>();
+const proxyToRawFlexNode = new WeakMap<TreeNode, RawTreeNode<FlexTreeNodeSchema, unknown>>();
```

Comment:
I think this was a typo. I see no reason this should have been field node specific.